wolfcrypt/src/logging.c

```diff
@@ -364,6 +364,27 @@ static void wolfssl_log(const int logLevel, const char *const logMessage)
 }
 
 #ifndef WOLFSSL_DEBUG_ERRORS_ONLY
+#ifdef __clang__
+/* tell clang argument 1 is format */
+__attribute__((__format__ (__printf__, 1, 0)))
+#endif
+#if !defined(_WIN32) && defined(XVSNPRINTF)
```

Comment:
It looks like types.h always defines `XVSNPRINTF`. Consider having another way to disable? Also if disabled should the format arg still be sent to `wolfssl_log`?